.eleventy.js

```diff
@@ -20,18 +20,36 @@ module.exports = function(config) {
   config.addFilter('dateFilter', dateFilter);
   config.addFilter('markdownFilter', markdownFilter);
   config.addFilter('w3DateFilter', w3DateFilter);
-  
+
 
   // Transforms
   config.addTransform('htmlmin', htmlMinTransform);
   config.addTransform('parse', parseTransform);
 
-   // Passthrough copy
-   config.addPassthroughCopy({"src/assets/images": "assets/images"});
-   config.addPassthroughCopy({"src/assets/js": "assets/js"});
-   config.addPassthroughCopy({"src/lib": "lib"});
-   config.addPassthroughCopy({"src/assets/stylesheets": "assets/stylesheets"});
-   config.addPassthroughCopy({"src/posts/images": "posts/images"})
+  // Passthrough copy
+  config.addPassthroughCopy({"src/assets/images": "assets/images"});
+  config.addPassthroughCopy({"src/assets/js": "assets/js"});
+  config.addPassthroughCopy({"src/assets/stylesheets": "assets/stylesheets"});
+  config.addPassthroughCopy({"src/posts/images": "posts/images"})
+  config.addPassthroughCopy({"node_modules/infusion/dist/infusion-all.min.js": "lib/infusion/infusion-uio.min.js"});
```

Comment:
I think you missed updating the left hand side of the passthrough configuration.
```suggestion
  config.addPassthroughCopy({"node_modules/infusion/dist/infusion-uio.min.js": "lib/infusion/infusion-uio.min.js"});
```